examples/python/multiprocessing/README.md

```diff
@@ -0,0 +1,51 @@
+## Multiprocessing with gRPC Python
+
+Multiprocessing allows application developers to sidestep the Python global
+interpreter lock and achieve true concurrency on multicore systems.
+Unfortunately, using multiprocessing and gRPC Python is not yet as simple as
+instantiating your server with a `futures.ProcessPoolExecutor`.
+
+The library is implemented as a C extension, maintaining much of the state that
+drives the system in native code. As such, upon calling
+[`fork`](http://man7.org/linux/man-pages/man2/fork.2.html), much of the
+state copied into the child process is invalid, leading to hangs and crashes.
+
+However, calling `fork` without `exec` in your python process is supported
+*before* any gRPC servers have been instantiated. Application developers can
+take advantage of this to parallelize their CPU-intensive operations.
+
+## Running the Example
+
+This example calculates the first 10,000 prime numbers as an RPC. We instantiate
+one server per subprocess, balancing requests between the servers using the
+[`SO_REUSEPORT`](https://lwn.net/Articles/542629/) socket option.
```

Comment:
Should we mention the `SO_REUSEPORT` is not supported by default in our `manylinux1` binary wheel?